backend/app/routers/order.py

```
@@ -6,69 +6,28 @@
 import app.models as models
 from app.database import get_db, SessionLocal
 from app.utils import get_current_user
+from app.helpers.orders import (
+    get_product_or_404,
+    get_order_or_404,
+    get_bill_or_404,
+    get_order_items_details,
+    get_user_brand_ids,
+    validate_artisan_order_access,
+    format_order_response,
```

Ruff (0.8.2)

`app.helpers.orders.format_order_response` imported but unused

Remove unused import: `app.helpers.orders.format_order_response`

(F401)